pandas/core/index.py

```diff
@@ -1677,33 +1690,52 @@ def get_indexer(self, target, method=None, limit=None):
 
         pself, ptarget = self._possibly_promote(target)
         if pself is not self or ptarget is not target:
-            return pself.get_indexer(ptarget, method=method, limit=limit)
+            return pself.get_indexer(ptarget, method=method, limit=limit,
+                                     tolerance=tolerance)
+
+        if tolerance is not None:
+            tolerance = self._convert_tolerance(tolerance)
```

Comment:
yeh I saw that. But I don't think this gets called if only ``get_indexer`` is called.